python/tensorrt_model_connect/families/cosmos3/vae_step_builder.py

```
@@ -0,0 +1,555 @@
+        raise RuntimeError(
+            f"Cosmos3 VAE step graph populated {cache_index} caches with "
+            f"{sum(value is not None for value in cache_outputs)} outputs, expected 32"
+        )
+    expected_video_shape = profile.video_shape(first_frame_only=first_frame_only)
+    if tuple(x.shape) != expected_video_shape:
+        raise RuntimeError(
+            f"Cosmos3 VAE step output is {tuple(x.shape)}, expected {expected_video_shape}"
+        )
+    x.name = "video_frame"
+    network.mark_output(x)
+    for spec, cache_output in zip(VAE_STEP_CACHE_SPECS, cache_outputs):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)